### DISCUSSION TITLE
Updates too Frequent

### DISCUSSION THREAD

**Discussion by pelaird** [turkaysoft/glow#22] [General]
I like this app, but updates every other day get very tedious. Maybe hold off on updates to once a week.

**Comment by turkaysoft** [comment]
> I like this app, but updates every other day get very tedious. Maybe hold off on updates to once a week.

Hello,

Thank you for your suggestion.
Updates are usually released after reaching a certain development number.

There are about 100 features that I want to bring to Glow and are in the development process.

I add them as I develop them and this naturally requires updates.
I don't think there is much point in waiting to release a feature that is ready.

However, as for your suggestion, I generally try to patch on Fridays every week.
